Modules/System/Barcode/src/Barcode Provider 2D/BarcodeEncodeSettings2D.Table.al

```diff
@@ -0,0 +1,54 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Common setting used when encoding @D barcodes.
```

Comment:
2D rather than @D? And are these settings indeed for 2D barcodes, or are these specific to the QR code symbology?

Reply:
Typo fixed.